docker-app/qfieldcloud/subscription/models.py

```diff
@@ -0,0 +1,80 @@
+from datetime import timedelta
+
+from django.core.validators import MinValueValidator
+from django.db import models
+
+
+class AccountType(models.Model):
+    @classmethod
+    def get_or_create_default(cls) -> "AccountType":
+        """Returns the default account type, creating one if none exists.
+        To be used as a default value for UserAccount.type"""
+        if cls.objects.count() == 0:
+            cls.objects.create(
+                code="default",
+                display_name="default (autocreated)",
+                is_default=True,
+                is_public=False,
+            )
+        return cls.objects.order_by("-is_default").first()
+
+    code = models.CharField(max_length=30, unique=True)
+    # TODO: match requirements in QF-234 (fields like automatic old versions)
+    # TODO: decide how to localize display_name. Possible approaches:
+    # - django-vinaigrette (never tried, but like the name, and seems to to exactly what we want)
+    # - django-modeltranslation (tried, works well, but maybe overkill as it creates new database columns for each locale)
+    # - something else ? there's probably some json based stuff
+    display_name = models.CharField(max_length=100)
+    storage_mb = models.PositiveIntegerField(default=10)
+    storage_keep_versions = models.PositiveIntegerField(default=10)
+    job_minutes = models.PositiveIntegerField(default=10)
+    can_add_storage = models.BooleanField(default=False)
+    can_add_job_minutes = models.BooleanField(default=False)
+    is_external_db_supported = models.BooleanField(default=False)
+    has_priority_support = models.BooleanField(default=False)
+    minimum_repackaging_time = models.DurationField(
+        default=timedelta(minutes=60),
+        validators=[MinValueValidator(timedelta(minutes=1))],
+    )
```

Comment:
```suggestion
    can_configure_repackaging_cache_expire = models.BooleanField(default=False)
    min_repackaging_cache_expire_m = models.DurationField(
        default=timedelta(minutes=60),
        validators=[MinValueValidator(timedelta(minutes=1))],
    )
```
Did you ommit the unit of measure on purpose? On clickup it was specified as `minimum_repackaging_time_m`, so it is clear we store minutes?


Giving a second thought, now the name in `projects` and here match. I felt a bit confused by the `repackaging_time`, it sounded like a time of day (e.g. 23:30).

Reply:
same as above for the units, ok for the rename